crud/common/utils.lua

```diff
@@ -691,4 +691,31 @@ function utils.check_name_isident(name)
     return true
 end
 
+--- Insert each value form values to list
+--
+-- @function table_extend
+--
+-- @param table list
+--  List to be extended
+--
+-- @param table values
+--  Values to be inserted to list
+--
+-- @return[1] list
+--  List with old values and inserted values
+function utils.table_extend(list, values)
+    dev_checks('table', 'table')
+
+    if values == nil then
+        return list
+    end
+
+    list = list or {}
+
+    return fun.reduce(
+        function(list, value) table.insert(list, value) return list end,
```

Comment:
Using function allocation in cycle seems like not the best idea.
```
    for _, result in pairs(results) do
        rows = utils.table_extend(rows, result[1])
    end
```

Reply:
Updated:

```lua
function utils.table_extend(list, values)
    dev_checks('table', 'table')

    for _, value in ipairs(values) do
        table.insert(list, value)
    end

    return list
end
```